### ISSUE TITLE
Package Specific Branch/Line coverage rate?

### ISSUE THREAD

**Issue by JoshCode** [cobertura/cobertura#215]
Hi!

I am trying to have Cobertura ignore a specific package in my project. I do want to test for code coverage, but I don't want Cobertura to fail the build if something is wrong with this package. I do want Cobertura to fail the build if the coverage is too low for the other packages.

**What I tried**
- Excluding the package and classes within in the `instrumentation` configuration, however this made Cobertura not go over them at all, what I don't want.
- Ignoring the package and classes within in the `instrumentation` configuration. I did not notice what change the `<ignores></ignores>` make.
- Setting the `branchRate` and `lineRate` in the `check` configuration to `0` with the following regex: `nl.joshuaslik.UFMReckoning.gui.*`, however this will still fail, because the `packageLineRate` and `packageBranchRate` are below the target value.

Does anybody know if it is possible what I am trying to achieve? If not, I think this would be a nice feature to have.

**TL;DR**

Is it possible to define a `branchRate` and `lineRate` for a specific package.
